multipaz/src/commonTest/kotlin/org/multipaz/sdjwt/SdJwtVcTest.kt

```diff
@@ -1,5 +1,6 @@
 package org.multipaz.sdjwt
 
+import io.ktor.utils.io.charsets.Charsets
```

Comment:
This looks unused? If not unused, it's probably not appropriate to use a ktor library function here.

Reply:
Done.